x-pack/plugins/stack_connectors/server/connector_types/stack/index.ts

```diff
@@ -49,6 +49,8 @@ export {
 } from './webhook';
 export type { ActionParamsType as WebhookActionParams } from './webhook';
 
+export { getOpsgenieConnectorType, OpsgenieConnectorTypeId } from './opsgenie';
```

Comment:
I'll add the `OpsgenieActionParams` in the next UI PR when it's actually needed externally